doc/waarp-r66/source/installation/installation.rst

```diff
@@ -8,9 +8,12 @@ Les prérequis pour WaarpR66 sont les suivants :
 
 * Un OS supporté : Linux (toutes distributions) et Windows 32 ou 64 bits
 * Java 1.6 minimum (java 1.8 recommandé)
-* Base de données PostgreSQL (version 9.4 minimum)
-
-Les interfaces web nécessitent un navigateur récent (Chrome, Firefox, Internet Explorer 10+).
+* Base de données PostgreSQL (version 9.4 minimum) est recommandée, MySQL,
+  MariaDB, Oracle SQL server et H2 sont supportés.
+* Les interfaces web nécessitent un navigateur récent (Chrome, Firefox, Internet
+  Explorer 10+).
+* 128Mo de RAM minimum pour un client R66 (512Mo recommandés), 512Mo de RAM
+  minimum pour un serveur (1Go recommandé)
```

Comment:
Maybe add a special notice for heavy loaded R66 server that could need more than 1GB, such as 2GB (for instance the tests with 5K concurrent files show that 2GB is ok while 1GB could be not enough), so the memory recommandation depends on the load on server side (client is not concerned) ?

Reply:
@bcarlin just a suggestion

Reply:
Good suggestion! I've edited the doc.